catalogue/webapp/components/WorkDetails/WorkDetails.js

```diff
@@ -74,6 +75,18 @@ const WorkDetails = ({
   itemsRef.current = itemsWithPhysicalLocations;
   const [hasRequestableItems, setHasRequestableItems] = useState(false);
   const singleItem = itemsWithPhysicalLocations.length === 1;
+  const [imageJSON, setImageJSON] = useState(null);
+  const fetchImageJSON = async () => {
+    try {
+      const imageJSON =
+        iiifImageLocation &&
+        (await fetch(iiifImageLocation.url).then(resp => resp.json()));
```

Comment:
I always think it's a bit strange mixing `await` and `.then`. Although I guess the double-`await` pattern is weird too.